tapiro/core/src/main/scala/io/buildo/tapiro/Util.scala

```diff
@@ -36,9 +33,11 @@ object TapiroRouteError {
 
 case class TapiroRoute(route: Route, error: TapiroRouteError)
 
-class Util(logger: ManagedLogger) {
+class Util() {
   import Formatter.format
 
+  val logger = LogManager.getLogger("io.buildo.tapiro")
```

Comment:
this does not show up when running the sbt plugin, I'll open a separate issue for it